src/main/java/org/jitsi/jigasi/JvbConference.java

```diff
@@ -1599,4 +1609,111 @@ public String toString()
                 + "]@"+ hashCode();
         }
     }
+
+    /**
+     * Sets the chatroom presence for the participant.
+     *
+     * @param muted <tt>true</tt> for presence as muted,
+     * false otherwise
+     */
+    void setChatRoomAudioMuted(boolean muted)
+    {
+        if (mucRoom != null)
+        {
+            AudioMutedExtension audioMutedExtension = new AudioMutedExtension();
+
+            audioMutedExtension.setAudioMuted(muted);
+
+            OperationSetJitsiMeetTools jitsiMeetTools
+                = xmppProvider.getOperationSet(OperationSetJitsiMeetTools.class);
+
+            jitsiMeetTools
+                .sendPresenceExtension(mucRoom, audioMutedExtension);
+
+        }
+    }
+
+    /**
+     * Request Jicofo on behalf of Jigasi to mute a participant.
+     *
+     * @param bMuted <tt>true</tt> if request is to mute audio,
+     * false otherwise
+     * @return <tt>true</tt> if request succeeded, false
+     * otherwise
+     */
+    public boolean requestAudioMute(boolean bMuted)
+    {
+        StanzaCollector collector = null;
+        try
+        {
+            String roomName = callContext.getRoomName();
+            if (!roomName.contains("@"))
+            {
+                // we check for optional muc service
+                String mucService
+                    = JigasiBundleActivator.getConfigurationService()
+                        .getString("org.jitsi.jigasi.MUC_SERVICE_ADDRESS", null);
+                if (!StringUtils.isNullOrEmpty(mucService))
+                {
+                    roomName = roomName + "@" + mucService;
+                }
+            }
+
```

Comment:
There should be an easier way to obtain the room address.

Reply:
What about mucRoom.getIdentifier()?

Reply:
Changed to getIdentifier.